src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java

```diff
@@ -70,12 +72,37 @@ public String claimantFullName() {
      * @return title according to conditions defined above
      */
     private String getTitle() {
-        return !Strings.isNullOrEmpty(trimStringValue(claimantTitle)) ? Other.equals(claimantTitle)
-                ? !Strings.isNullOrEmpty(claimantTitleOther)
-                ? claimantTitleOther : "" : claimantTitle
-                : !Strings.isNullOrEmpty(trimStringValue(claimantPreferredTitle))
-                ? Other.equals(claimantPreferredTitle) ? !Strings.isNullOrEmpty(claimantTitleOther)
-                ? claimantTitleOther : "" : claimantPreferredTitle : "";
+        //Title is set as expected
+        if (!Strings.isNullOrEmpty(trimStringValue(claimantTitle))
+            && !OTHER.equals(trimStringValue(claimantTitle))
+            && (!PREFER_NOT_TO_SAY.equals(trimStringValue(claimantTitle)))) {
+            return claimantTitle;
+        }
+
+        //If title is Other & custom title is added
+        if (OTHER.equals(trimStringValue(claimantTitle))
+            && (!Strings.isNullOrEmpty(claimantTitleOther))) {
+            return claimantTitleOther;
+        }
+
+        //Preferred title is set rather than the normal title
+        if (Strings.isNullOrEmpty(trimStringValue(claimantTitle))
+            && OTHER.equals(trimStringValue(claimantPreferredTitle))) {
+            return claimantTitleOther;
+        }
+
+        //If title is "Prefer Not to Say"
+        if (PREFER_NOT_TO_SAY.equals(trimStringValue(claimantTitle))) {
```

Comment:
claimant preferred title will be prefer not to say and not claimantTitle

Reply:
Please have all the checks for field claimantPreferredTitle too

Reply:
About the first comment: From the screen shot on this ticket,  it is claimantTitle can be "Prefer not to say". Can both be "Prefer not to say"? Are they not exclusive? Bt that I mean they do not get set both at the same time.  

About the second comment: I check the conditions for claimantPreferredTitle too.

Reply:
I add fix as discussed.